src/controllers/SigninController.ts

```diff
@@ -0,0 +1,41 @@
+import SigninDao from "../daos/Mongo/SigninDao";
+import { SessionKey } from "ch-node-session-handler/lib/session/keys/SessionKey";
+import { SignInInfoKeys } from "ch-node-session-handler/lib/session/keys/SignInInfoKeys";
+
+class SigninController {
```

Comment:
I think it may be worth putting a comment in this file to say that it won't be used in live. Especially as it has plain text password matching in it.